app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/BrowserNavigationOverlay.kt

```diff
@@ -204,9 +205,12 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
 
         val manager = HomeTileManager(context, COL_COUNT)
         manager.spanSizeLookup = object : GridLayoutManager.SpanSizeLookup() {
+
+            val localeIsEnUs = LocaleManager.getInstance().isLocaleENUS(context)
```

Comment:
Nit: this is slightly inconsistent with the other localeIsENUS - I don't care which it is, but I think they should be consistent. (But you're right, pretty much all of these are terrible.)